CMakeLists.txt

```diff
@@ -634,9 +634,8 @@ if ( USE_STATIC_LIBS )
     endif()
 endif()
 
-
 # zlib
-if (BUILD_CLIENT OR BUILD_TTY_CLIENT OR BUILD_SERVER OR BUILD_DUMMY_APP OR NACL)
+if (BUILD_CLIENT OR BUILD_TTY_CLIENT OR BUILD_SERVER OR BUILD_DUMMY_APP OR (NACL AND BUILD_CGAME))
```

Comment:
If we don't need zlib condition could be updated to exclude Saigo builds

Reply:
Done.